.github/workflows/ci.yml

```
@@ -9,6 +9,7 @@ on:
     paths-ignore:
       - "general-superstaq/general_superstaq/_version.py"
   pull_request:
+    types: [review_requested, ready_for_review]
```

Do we have to specifiy
```suggestion
    types: [opened, synchronize, reopened, review_requested, ready_for_review]
```
also? Or are those assumed by default?